recipes/glassure/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "glassure" %}
+{% set version = "1.4.2" %}
```

Comment:
Can you update it to the newest version and then we try again? I cannot see the issue right now.